schemas/api-reference.json

```diff
@@ -64,6 +64,15 @@
             "type":           "string",
             "description":    "Name of the `function` this is a stable identifier for use in auto-generated client libraries"
           },
+          "stability": {
+            "title":          "Stability-Level",
```

Comment:
It would be nice to add "type": "string" here as this is easily interpreted by code generators such as that used in taskcluster-client-go and taskcluster-client-java. It allows the type to be inferred so it will not default to interface{} but string (go) and not Object but String (java). In this case it is in api-reference.json rather than an input/output schema so is not so critical (I don't interpret the api-reference.json but it would be nice to have in case that changes).